packages/mdc-side-sheet/_mixins.scss

```diff
@@ -0,0 +1,216 @@
+//
+// Copyright 2017 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "@material/ripple/mixins";
+@import "@material/shape/mixins";
+@import "@material/theme/mixins";
+@import "./variables";
+
+//
+// Public
+//
+
+@mixin mdc-side-sheet-stroke-color($color, $opacity: $mdc-side-sheet-stroke-opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  @include mdc-theme-prop(border-color, $value);
+}
+
+@mixin mdc-side-sheet-shape-radius($radius, $side: trailing) {
```

Comment:
Nit: We should be consistent on whether `leading` and `trailing` are quoted or unquoted. Currently we have the default unquoted but everything else seems to be quoted. I think we do unquoted elsewhere in MDC but I'd have to dig for examples.

(Fortunately Sass treats the two as equal, though I wonder if that would ever change...)

Reply:
Removed double quotes everywhere